.github/workflows/generate-rst.yml

```diff
@@ -0,0 +1,39 @@
+name: Generate cells docs
+
+on:
+    push:
+    pull_request:
+
+jobs:
+
+  generate-cells:
+    runs-on: ubuntu-18.04
+    steps:
+
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Generate cells READMEs
+        run: |
+          sudo apt update
+          sudo apt install -y python3 python3-pip git tar gzip
+          pip3 install docutils sphinx
+
+          SUBMODULE_VERSION=latest make submodules -j3 || make submodules -j1
```

Comment:
`git submodule update --init --recursive` ?

Reply:
`git submodule update --init --recursive` will fetch all the submodules, and we need only the `latest` ones